source/brailleDisplayDrivers/dotPad.py

```diff
@@ -0,0 +1,475 @@
+# A part of NonVisual Desktop Access (NVDA)
```

Comment:
Is it possible to structure this like `source\brailleDisplayDrivers\albatross` or `source\brailleDisplayDrivers\eurobraille`?